util/blockdevice/util.go

```diff
@@ -88,7 +91,70 @@ func GetCapacityOrError(obj unstructured.Unstructured) (resource.Quantity, error
 			errors.Errorf("Can not get capacity: Expected kind %q got %q",
 				types.KindBlockDevice, obj.GetKind())
 	}
-	return unstruct.GetQuantityOrError(&obj, "spec", "capacity", "storage")
```

Comment:
Please provide details of error due to GetQuantityOrError in PR message.

Reply:
done